lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -506,6 +513,21 @@ class DetailsRenderer {
     if (item.selector) element.setAttribute('data-selector', item.selector);
     if (item.snippet) element.setAttribute('data-snippet', item.snippet);
 
+    if (!item.boundingRect || !this._fullPageScreenshot) {
+      return element;
+    }
+
+    const fullPageScreenshot = this._fullPageScreenshot;
+
+    const elementScreenshot = ElementScreenshotRenderer.render(
```

Comment:
feels like we usually create an instance with the `_dom` and all that jazz. I never really loved the pattern but this feels out of place.

Reply:
crc?